### PR TITLE
3.0 release

### PR DESCRIPTION
**Pull request by MalinAhlberg**
In this major release we provide access to the newly created database <i>GotEnzymes</i> (article under consideration). GotEnzymes contains over 25.7 million predicted enzyme turnover numbers (k<sub>cat</sub>) for +5.8 million unique sequences and 4147 compounds, across 8099 species. The values are predicted by cutting-edge artificial intelligence tools. These new data are stored in a newly-added Postgres database that is seamlessly integrated into the software stack. In addition, Metabolic Atlas has obtained an _Identifiers.org_ profile and has improved the _About_ pages.
<br> More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/3.0' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>.

**The work on GotEnzymes has been divided across the following issues:**
- #1023
- #1015
- #1014
- #1041 
- #1024
- #1032
- #1026
- #1039
- #1034
- #1027
- #1060
- #1065
- #1061
- #1086

**Other Metabolic Atlas:**
-  Issue: obtain profile on [identifiers.org](https://identifiers.org)
- #907
- #916
- #1035
- #1063

**Project maintenance**
- #1022
- #963
- #1049
- [Improve documentation about 3D network viewer](https://github.com/MetabolicAtlas/3d-network-viewer/issues/38)

### CODE REVIEW

File: frontend/src/content/news.json [hunk 1]
```diff
@@ -1,9 +1,14 @@
 {
   "2022": [
+    {
+      "date": "2022-06-30",
+      "title": "Metabolic Atlas 3.0",
+      "text": "<b>Metabolic Atlas 3.0</b> is out! In this major release we provide access to the newly created database <i>GotEnzymes</i>. This contains over 25,794,195 million predicted k<sub>cat</sub> from 8,099 species. The values are predicted by cutting-edge artificial intelligence tools. For more information the prediction methods, please have a look at <a href='https://doi.org/10.1038/s41929-022-00798-z'><i>Deep learning-based k<sub>cat</sub> prediction enables improved enzyme-constrained model reconstruction</i></a>. <br> More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/3.0' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
```
Comment (nanjiangshu) [inline frontend/src/content/news.json:6]: I would suggest 
1. `by cutting-edge artificial intelligence tools` => `by the cutting-edge artificial intelligence tools`
2. `For more information the prediction methods` => `For more information of the prediction methods`

Comment (mihai-sysbio) [reply frontend/src/content/news.json:6]: I don't think the extra `the` is an improvement for (1).
For (20), my suggestion is slightly different: `For more information *about* the prediction methods`.

Comment (MalinAhlberg) [reply frontend/src/content/news.json]: An "about" is added in 9e3032a.
Since there seems to be a majority against the extra "the", I did not include it.

**Comment by nanjiangshu** [comment]
@MalinAhlberg, regarding the description
1. Add also **about** in the sentence `For more information the prediction methods`
2.  For the sentence `This contains over 25,794,195 million predicted kcat...`, I guess `million` is not needed here.

**Comment by MalinAhlberg** [comment]
> @MalinAhlberg, regarding the description
> 
> 1. Add also **about** in the sentence `For more information the prediction methods`
> 2. For the sentence `This contains over 25,794,195 million predicted kcat...`, I guess `million` is not needed here. otherwise the number seems to be too big, please verify @mihai-sysbio.  Check also the text the file `news.json`
> 3. Since the dataset has been updated since June, please also verify all the numbers listed here.

Thanks for the input @nanjiangshu! 1. and 2. are fixed in the description and in `news.json`. 2. still exists on the webpage though (see https://dev-metatlas.csbi.chalmers.se/gotenzymes). Is this something we should fix before tomorrow @mihai-sysbio ? 
Regarding 3., do we have any better numbers at hand @mihai-sysbio ?

**Comment by inghylt** [comment]
I'm waiting to approve until we know that numbers etc. are correct, but I think everything looks good.

```
postgres=# SELECT COUNT(*) FROM enzymes;
  count   
----------
 33548884
(1 row)
```

This is what I get on the latest version of the input data. Talked to @e0 about it and we agree on that it makes sense to try to get the researchers input on this

**Comment by mihai-sysbio** [comment]
> I'm waiting to approve until we know that numbers etc. are correct, but I think everything looks good.
> 
> ```
> postgres=# SELECT COUNT(*) FROM enzymes;
>   count   
> ----------
>  33548884
> (1 row)
> ```
> 
> This is what I get on the latest version of the input data. Talked to @e0 about it and we agree on that it makes sense to try to get the researchers input on this

The new number should be 
```
select count(distinct(organism, gene, compound)) from enzymes;
count: 25795560
```

File: frontend/src/content/news.json [hunk 2]
```diff
@@ -1,9 +1,14 @@
 {
   "2022": [
+    {
+      "date": "2022-06-30",
```
Comment (mihai-sysbio) [inline frontend/src/content/news.json:4]: The date should be updated to today:
```suggestion
      "date": "2022-08-12",
```

File: frontend/src/content/news.json [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
+      "title": "Metabolic Atlas 3.0",
```
Comment (mihai-sysbio) [inline frontend/src/content/news.json:5]: I'd like to also mention GotEnzymes in the title:
```suggestion
      "title": "Metabolic Atlas 3.0 presents GotEnzymes",
```

File: frontend/src/content/news.json [hunk 4]
```diff
@@ -1,9 +1,14 @@
...
+      "text": "<b>Metabolic Atlas 3.0</b> is out! In this major release we provide access to the newly created database <i>GotEnzymes</i>. This contains over 25,794,195 predicted k<sub>cat</sub> from 8,099 species. The values are predicted by cutting-edge artificial intelligence tools. For more information about the prediction methods, please have a look at <a href='https://doi.org/10.1038/s41929-022-00798-z'><i>Deep learning-based k<sub>cat</sub> prediction enables improved enzyme-constrained model reconstruction</i></a>. <br> More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/3.0' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
```
Comment (mihai-sysbio) [inline frontend/src/content/news.json:6]:
```suggestion
      "text": "We are delighted to announce the official release of <b>Metabolic Atlas 3.0</b> 🎉 In this major release we provide access to the newly created database <i>GotEnzymes</i> (article under consideration). GotEnzymes contains over 25.7 million predicted enzyme turnover numbers (k<sub>cat</sub>) for +5.8 million unique sequences and 4147 compounds, across 8099 species. The values are predicted by cutting-edge artificial intelligence tools. In addition, Metabolic Atlas has obtained an _Identifiers.org_ profile and has improved the _About_ pages.
<br> More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/3.0' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>.
."
```

**Review by mihai-sysbio — CHANGES_REQUESTED** [review]
This looks great! I've left some suggestion for the news, and also took the liberty to update the PR body, including regrouping some issues.

_Note_: we have a last minute PR to be added #1085

File: frontend/src/content/news.json [hunk 5]
```diff
@@ -1,9 +1,16 @@
 {
   "2022": [
+    {
+      "date": "2022-08-12",
+      "title": "Metabolic Atlas 3.0 presents GotEnzymes",
+      "text": "We are delighted to announce the official release of <b>Metabolic Atlas 3.0</b> 🎉 In this major release we provide access to the newly created database <i>GotEnzymes</i> (article under consideration). GotEnzymes contains over 25.7 million predicted enzyme turnover numbers (k<sub>cat</sub>) for +5.8 million unique sequences and 4147 compounds, across 8099 species. The values are predicted by cutting-edge artificial intelligence tools. In addition, Metabolic Atlas has obtained an _Identifiers.org_ profile and has improved the _About_ pages.
```
Comment (mihai-sysbio) [inline frontend/src/content/news.json]: It looks like I previously copied over some GH markdown:

```suggestion
      "text": "We are delighted to announce the official release of <b>Metabolic Atlas 3.0</b> 🎉 In this major release we provide access to the newly created database <i>GotEnzymes</i> (article under consideration). GotEnzymes contains over 25.7 million predicted enzyme turnover numbers (k<sub>cat</sub>) for +5.8 million unique sequences and 4147 compounds, across 8099 species. The values are predicted by cutting-edge artificial intelligence tools. In addition, Metabolic Atlas has obtained an <i>Identifiers.org</i> profile and has improved the <i>About</i> pages.
```

**Review by mihai-sysbio — APPROVED** [review]
🥁 🎺

**Review by nanjiangshu — APPROVED** [review]
All good!

**Pull request merged**